requirements/requirements_dev.txt

```diff
@@ -1,3 +1,4 @@
+build
```

Comment:
do we need this to build the rpm, or just for dev env?

Reply:
This is only required for building the sdist, which we only do for the upstream builds.